src/DistributedComputations/distributed_grids.jl

```diff
@@ -41,7 +41,7 @@ end
 @inline local_sizes(N, R::Fractional) = Tuple(ceil(Int, N * r) for r in R.sizes)
 @inline function local_sizes(N, R::Sizes)
     if N != sum(R.sizes)
-        @warn "The domain size specified in the architecture $(R.sizes) is inconsistent 
+        @warn "The domain size specified in the architecture $(sum(R.sizes)) is inconsistent 
```

Comment:
Do we really specify `sum(R.sizes)` when we build the architecture?

Reply:
The error message has to repeat back what the user put in to be understandable I think, but its ok if it includes extra info like `sum(R.sizes)`